backend/src/test/java/com/github/chuettenrauch/mixifyapi/integration/user/controller/UserControllerTest.java

```diff
@@ -56,6 +59,25 @@ void me_returnsUserResourceIfLoggedIn() throws Exception {
                 .andExpect(content().json(expectedJson, true));
     }
 
+    @Test
+    void logout_returnsUnauthorizedIfNotLoggedIn() throws Exception {
```

Comment:
Naming convention for test would be better when done like: "logout_whenX_thenY" or "logout_whenX_shouldY"

[https://stackoverflow.com/questions/96297/what-are-some-popular-naming-conventions-for-unit-tests](https://stackoverflow.com/questions/96297/what-are-some-popular-naming-conventions-for-unit-tests)

[https://dzone.com/articles/7-popular-unit-test-naming](https://dzone.com/articles/7-popular-unit-test-naming)